project/src/store/user-process/selectors.ts

```diff
@@ -5,3 +5,11 @@ export const getAuthorizationStatus = (state: State): AuthorizationStatus =>
   state[NameSpace.User].authorizationStatus;
 export const getAuthCheckedStatus = (state: State): boolean =>
   state[NameSpace.User].authorizationStatus !== AuthorizationStatus.Unknown;
+
+export const getAuthLogInStatus = (state: State): boolean =>
+  state[NameSpace.User].authorizationStatus === AuthorizationStatus.Auth;
+
+export const getUserEmail = (state: State): string =>
+  state[NameSpace.User].userData.email;
+export const getAvatarUrl = (state: State): string =>
+  state[NameSpace.User].userData.avatarUrl;
```

Comment:
В дополнение к комментарию который я писал выше:

```suggestion
export const getUserData = (state: State) => state[NameSpace.User].userData;
```